<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Fawcett App</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600; color:#79b656;&quot;&gt;Fawcett App &lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#79b656;&quot;&gt;(v4)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="intro_text">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Create a quick proof of the 'Questions Tabled On' document. First select the tableing date for the questions then press 'Create Quick Proof'. This program will query the &lt;a href=&quot;https://api.eqm.parliament.uk/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#419cff;&quot;&gt;EQM API&lt;/span&gt;&lt;/a&gt; for the relevant questions and arrange them for you in a web browser. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="date_label">
         <property name="text">
          <string>Select tableing date</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDateEdit" name="dateEdit">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QPushButton" name="create_proof_btn">
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Create Quick Proof</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="QPushButton" name="logBtn">
      <property name="text">
       <string>Open Log</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Usage Instructions, background and installation instructions are avaliable on &lt;a href=&quot;https://hopuk.sharepoint.com/:w:/r/sites/bct-ppu/_layouts/15/Doc.aspx?sourcedoc=%7B6B4ACE86-8B55-4742-A8EC-12BE078E67A2%7D&amp;amp;file=FawcettAppInstructions.docx&amp;amp;action=default&amp;amp;mobileredirect=true&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#419cff;&quot;&gt;SharePoint.&lt;/span&gt;&lt;/a&gt;&lt;br/&gt;You can also view the source code on &lt;a href=&quot;https://github.com/hoc-ppu/Fawcett&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#419cff;&quot;&gt;GitHub&lt;/span&gt;&lt;/a&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionInstilation_Instructions">
   <property name="text">
    <string>Instilation Instructions</string>
   </property>
  </action>
  <action name="actionView_source_code">
   <property name="text">
    <string>View source code</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
